src/content/components/ActiveRSMessages/ActiveRSMessages.tsx

```diff
@@ -149,7 +151,17 @@ export class ActiveRSMessages extends React.PureComponent {
       })
       .then(messages =>
         this.setState({ messages, selectedBucket: BUCKETS[bkey] })
-      );
+      )
+      .catch(() => {
+        ReactDOM.render(
+          <ErrorView
+            header={"Error"}
+            subheader={"There was an error fetching data."}
+            buttonText={"Try again"}
+          />,
+          document.getElementById("root")
+        );
```

Comment:
Hmmm... subcomponents shouldn't be taking over rendering. Really, `ReactDOM.render` should only ever be called once. Ideally, if there's a catastrophic error, the subcomponent should navigate via the router. One way to do that would be setting an `errored` state, and if true, the render method should render a [Redirect](https://v5.reactrouter.com/web/api/Redirect) to the error page.

Though it will take some effort and research on the react router docs to determine the best way to pass the necessary parameters to the error route (like what location the try again button should navigate to). I'm sure there are lots of blog posts and discussions on how to do an error page with react router, so we could figure it out.

This also requires testing to make sure we don't break the site, so I would be fine with just blowing off this issue for now. These bugs (and the string bugs they each depend on) are much higher priority:
- https://bugzilla.mozilla.org/show_bug.cgi?id=1828170
- https://bugzilla.mozilla.org/show_bug.cgi?id=1828172